test/apps/jest-preset.js

```diff
@@ -0,0 +1,14 @@
+const path = require( 'path' );
+const base = require( '@automattic/calypso-jest' );
+
+module.exports = {
+	...base,
+	cacheDirectory: path.join( __dirname, '../../.cache/jest' ),
+	testEnvironment: 'jsdom',
+	transformIgnorePatterns: [
+		'node_modules[\\/\\\\](?!.*\\.(?:gif|jpg|jpeg|png|svg|scss|sass|css)$)',
+	],
+	setupFiles: [ 'jest-canvas-mock' ],
+	// This includes a lot of globals that don't exist, like fetch, matchMedia, etc.
+	setupFilesAfterEnv: [ require.resolve( '../client/setup-test-framework.js' ) ],
```

Comment:
I was looked into this, but we'd need to duplicate at least a few things from that file. From what I could tell, none of it was specific enough to the client setup to justify splitting it out. 

I think it'd be good to keep this in mind in the future though. I personally like having this shared as much as possible, since there aren't big differences in the environments! Both the client and ETK run in a browser, after all.